src/NftDetails/NftDetails.tsx

```diff
@@ -1,36 +1,48 @@
-import {useNavigation, useRoute} from '@react-navigation/native'
-import React, {useEffect, useState} from 'react'
+import {useNavigation} from '@react-navigation/native'
+import React, {ReactNode, useMemo, useState} from 'react'
+import {defineMessages, useIntl} from 'react-intl'
 import {Image, StyleSheet, TouchableOpacity, View} from 'react-native'
 import {ScrollView} from 'react-native-gesture-handler'
 
-import {CopyButton, FadeIn, OfflineBanner, Spacer, StatusBar, Text} from '../components'
+import {getPrettyDate} from '../../tests/helpers/utils'
+import {CopyButton, FadeIn, Icon, Link, OfflineBanner, Spacer, StatusBar, Text} from '../components'
 import {Tab, TabPanel, TabPanels, Tabs} from '../components/Tabs'
-import {useNfts} from '../hooks'
+import {useNfts, useTransactionInfos} from '../hooks'
+import {getAssetFingerprint} from '../legacy/format'
 import {NftDetailsNavigation} from '../navigation'
 import {useSelectedWallet} from '../SelectedWallet'
+import {COLORS} from '../theme'
 
-enum VIEW_TABS {
-  OVERVIEW = 'OVERVIEW',
-  METADATA = 'METADATA',
-}
+type VIEW_TABS = 'overview' | 'metadata'
 
-type Params = {id: string}
+type Props = {route: {params: {id: string}}}
```

Comment:
you can type with `StackScreenProps<ParamList...>`[ more ref](https://reactnavigation.org/docs/typescript/#organizing-types)

Reply:
also leveraging `useRoute`/`useParams`